js/shopping-list-item.js

```diff
@@ -0,0 +1,28 @@
+"use strict";
+
+class ShoppingListItem {
+  constructor(name, description) {
+    this.name = name;
+    this.description = description;
+    this.isDone = false;
+  }
+
+  check() {
+    return (this.isDone = true);
+  }
+
+  uncheck() {
+    return (this.isDone = false);
+  }
+
+  render(idx) {
+    if (this.isDone === true) {
+      return `<li class="completed_${this.isDone}"><input type="checkbox" onchange="changeCheckedStatus(${idx}, this)" checked/><span>${this.name}</span> <span>${this.description}</span></li>`;
```

Comment:
This could be done with just one string template literal if you made the `checked` string a variable and only added it if this shopping list item is done. Otherwise, just make the variable an empty string.